### ISSUE TITLE
Issues with mysql:8.0.36-debian

### ISSUE THREAD

**Issue by anatol06** [docker-library/mysql#1043]
Hi!
I have issues with mysql:8.0.36-debian image in Plesk Obsidian 18.0.60 and I am not sure, is it related with the latest Plesk update about 6 days ago or with latest Docker image push 6 days ago.
The problem is described there:
https://talk.plesk.com/threads/docker-problems-in-plesk-obsidian-18-0-60.374046/

**Comment by tianon** [comment]
https://github.com/docker-library/mysql/issues/303#issuecomment-822684258

**Comment by anatol06** [comment]
I installed the image mysql:8.0.36-debian from Docker Hub in Plesk Obsidian. 
The question is how to add cap_add: [ SYS_NICE ] to docker-compose.yml. Where can I find this file? I could not find it. i think, this should be added to the image before installing it

**Comment by tianon** [comment]
`SYS_NICE` is a capability that has to be added at runtime; we cannot add it to the image.  The issues you're having with the image are environmental, not something we can fix in the image.

(Also, the one about the IP address is simply a warning, and the `mbind` errors are something that, per the thread I linked, can be ignored if you don't mind the performance hit.)

**Comment by anatol06** [comment]
Thanks for the information! It's good at least that the websites are unaffected :)
However, I'd like to understand why everything was working fine a week ago and then encountered these issues:

Firstly, the logs disappeared, but they reappeared after I recreated the image in Plesk. This implies that the image I'm currently using is not the same as the one I had before. The image I'm using now was pushed on April 10th. Notably, Plesk was updated on April 11th, after which these problems surfaced.

Secondly, there's an issue with performing MySQL dumps for a large database of 22GB. I consistently encounter Error 2013 every time I attempt this. Despite trying various solutions found for this error, none have proven effective. I've spent the last three days troubleshooting these issues without success.

In summary: After Plesk was updated, I encountered problems with mysql:8.0.36-debian, including missing logs and failed backups due to lost connection to the MySQL server during queries (Error 2013). Although recreating the image in Plesk resolved the log issue, it introduced two new problems that were not present previously. This led me to discover that the image itself was also updated.

Now, I'm uncertain whether I should wait for another Plesk update or another image update in hopes that these issues will be resolved. Could you please assist in reaching out to Plesk to collaboratively identify and resolve these problems?

**Comment by tianon** [comment]
I'm sorry, but that's not something we're going to do.  It sounds like your Plesk update probably did something to change your deployment (or you got really unlucky in a combination of things).  I would suggest contacting Plesk or finding a consulting firm willing to help you resolve this.
